node/subsystem-util/src/lib.rs

```diff
@@ -394,13 +394,10 @@ impl Validator {
 
 /// ToJob is expected to be an enum declaring the set of messages of interest to a particular job.
 ///
-/// Normally, this will be some subset of `Allmessages`, and a `Stop` variant.
+/// Normally, this will be some subset of `Allmessages`.
 pub trait ToJobTrait: TryFrom<AllMessages> {
-	/// The `Stop` variant of the ToJob enum.
-	const STOP: Self;
-
-	/// If the message variant contains its relay parent, return it here
-	fn relay_parent(&self) -> Option<Hash>;
+	/// Get the relay parent this message is assigned to.
+	fn relay_parent(&self) -> Hash;
```

Comment:
This makes `ToJobTrait` a bit more rigid: it ensures that we'll never write a subsystem using this util which can have jobs not tied to relay parents.

I think this is a reasonable simplification at this point.